templates/plugins/jscomments/hypercomments.html.twig

```diff
@@ -0,0 +1,30 @@
+{#
+    HyperComments - IntenseDebate is a feature-rich comment system for
+    WordPress, Blogger, Tumblr and many other blogging/CMS platforms.
+
+    @see https://www.hypercomments.com/
+ #}
+<div id="jscomments">
+    <div id="hypercomments_widget"></div>
+
+    <script type="text/javascript">
+        _hcwp = window._hcwp || [];
+        _hcwp.push({
+            widget: "Stream",
+            widget_id: '{{ widget_id|e('js') }}',
+            social: '{{ social|e('js') }}'
+        });
+        (function () {
+            if ("HC_LOAD_INIT" in window)return;
```

Comment:
@Neiromaster The logic is the same. Only the curly braces are added that help readability. You can copy and paste it and add it to your PR.